caffe2/ideep/operators/conv_fusion_op.cc

```diff
@@ -127,6 +124,10 @@ class IDEEPConvFusionOp final : public IDEEPConvPoolOpBase {
   }
 
  private:
+  iprop pk_;
```

Comment:
Better name to show what this is? And please add comments to document these variables.

Reply:
Sure. Will describe the usage of all types introduced by ideep in ideep_utils.h.